### ISSUE TITLE
Editor import won't import included images

### ISSUE THREAD

**Issue by GiantCrocodile** [getkirby/editor#168]
Example of a line in an imported file:
`![image-123](C:\Users\<username>\[...]\image-123.png)`

In this case, the file is imported with a fixed absolute path. The referenced file is not uploaded.

**Comment by distantnative** [comment]
I don’t think a browser is able to just upload a file from someone’s computer by referencing a local path. That would be quite a security issue

**Comment by GiantCrocodile** [comment]
You are right @distantnative. Regardless of that, we still need a solution for that. An easy fix would be to remove the path and insert the typical 'upload image placeholder' thingies but maybe we find a better solution.
